ert_logging/logger.conf

```diff
@@ -26,6 +26,10 @@ handlers:
     formatter: simple_with_threading
     filename: ee-log.txt
 loggers:
+  job_runner.reporting:
```

Comment:
What is the main purpose of the new logging handle?

Reply:
I assume since job_runner module is a standalone top module, similar to ert_shared, it would need it's own logger. but if that is not the case it will be removed.

Reply:
the logging handle is removed